controllers/ovndbcluster_controller.go

```diff
@@ -581,11 +586,15 @@ func (r *OVNDBClusterReconciler) reconcileServices(
 	}
 
 	// Delete any extra services left after scale down
+	normalServiceLabels := map[string]string{
+		common.AppSelector: serviceName,
+		"type":             "normal",
```

Comment:
Not a big fan of the "normal" name, but the other name that I thought was "service" but can be "confusing" since both Headless and "normal" service are both services. I'm open to new namings